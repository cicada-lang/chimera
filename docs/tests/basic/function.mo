print () => {
  return 1
}

print (x) => {
  let y = x
  return y
}

// id

let id = (x) => {
  let y = x
  return y
}

print id(1)
print id(id(1))
print id(id(id(1)))

// zero arity

let one = () => {
  return 1
}

print one()
