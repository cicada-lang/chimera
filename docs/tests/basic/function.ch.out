() => {
  return 1
}
(x) => {
  let y = x
  return y
}
1
1
1
1
